src/cgtop/cgtop.c

```diff
@@ -916,6 +916,7 @@ static int run(int argc, char *argv[]) {
         usec_t last_refresh = 0;
         bool quit = false, immediate_refresh = false;
         _cleanup_free_ char *root = NULL;
+        typeof(arg_count) possible_count;
```

Comment:
Please typedef the enum and explicitly use it.

Reply:
changes in the force push.